roofit/roofitcore/res/RooNLLVarNew.h

```diff
@@ -26,6 +26,11 @@ namespace Experimental {
 class RooNLLVarNew : public RooAbsReal {
 
 public:
+   // The names for the weight variables that the RooNLLVarNew expects
+   static constexpr auto weightVarName = "_weight";
+   static constexpr auto weightVarNameSumW2 = "_weight_sumW2";
+   static constexpr auto weightVarNameSumW2Suffix = "_sumW2";
```

Comment:
This breaks a `Debug` build because `RooFit::BatchModeHelpers::createNLL` passes a `const char *` to the constructor of `RooRealVar`, but `constexpr` variables don't have an address.

Reply:
How can a variable not have an address? Sorry, I don't understand the problem here. Can you elaborate please? The code in `createNLL` is working, what part of it would not work in a debug build?

Reply:
Ah I see you have opened another PR for this, thanks!